internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -189,6 +193,12 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 	}
 	logger.WithField("name", kopsMetadata.Name).Info("Successfully updated storage class")
 
+	iamRole := fmt.Sprintf("nodes.%s-kops.k8s.local", cluster.ID)
```

Comment:
```suggestion
	iamRole := fmt.Sprintf("nodes.%s", kopsMetadata.Name)
```

The kops metadata name value is the full ID plus the `-kops.k8s.local` suffix. We should use that in case we ever decide to update how kops names are generated.

Reply:
Awesome, changing that thanks.